unified_targets/CONFIGS/HGLRCF745.config

```diff
@@ -0,0 +1,230 @@
+# Betaflight / STM32F745 (S745) 4.0.0 Apr  3 2019 / 14:32:23 (22b9f3453) MSP API: 1.41
+
+# start the command batch
+batch start
+
+board_name HGLRCF745
+manufacturer_id HGL
+
+# resources
+resource BEEPER 1 D15
+resource MOTOR 1 B00
+resource MOTOR 2 B01
+resource MOTOR 3 E09
+resource MOTOR 4 E11
+resource MOTOR 5 NONE
```

Comment:
Please remove these `NONE` lines and the rest of the lines that are defaults. Like I said before, you **need to follow the instructions:** https://github.com/betaflight/betaflight/blob/master/unified_targets/docs/CreatingAUnifiedTarget.md

Reply:
Sorry, I will modify it right away.